homeassistant/components/flexit_bacnet/number.py

```diff
@@ -23,10 +23,31 @@
 from .const import DOMAIN
 from .entity import FlexitEntity
 
+_MAX_FAN_SETPOINT = 100
+_MIN_FAN_SETPOINT = 30
+
 
 @dataclass(kw_only=True, frozen=True)
 class FlexitNumberEntityDescription(NumberEntityDescription):
-    """Describes a Flexit number entity."""
+    """Describes a Flexit number entity.
+
+    Setpoints for Away, Home and High are dependent of each other. Fireplace and Cooker Hood
+    have setpoints between 0 (MIN_FAN_SETPOINT) and 100 (MAX_FAN_SETPOINT).
+    See the table below for all the setpoints.
```

Comment:
I totally understand why you want this stored somewhere, but I don't think it should be in the `docstring`?.
It won't make sense anymore if you add number entities that are not linked to the setpoints.

I think it can be a regular comment instead

```python
    native_min_value_fn: Callable[[FlexitBACnet], int]
    set_native_value_fn: Callable[[FlexitBACnet], Callable[[int], Awaitable[None]]]

# Setpoints for Away, Home and High are dependent of each other. Fireplace and Cooker Hood
# have setpoints between 0 (MIN_FAN_SETPOINT) and 100 (MAX_FAN_SETPOINT).
# ...
#

NUMBERS: tuple[FlexitNumberEntityDescription, ...] = (
```

Reply:
Done!